zulipterminal/core.py

```diff
@@ -91,9 +91,10 @@ def show_help(self) -> None:
                           title="Help Menu (up/down scrolls)"),
             self.view,
             align='center',
-            width=help_view.width+2,  # +2 from LineBox
             valign='middle',
-            height=rows//2
+            # +2 to both of the following, due to LineBox
+            width=help_view.width+2,
+            height=min(3*rows//4, help_view.number_of_actions)+2
```

Comment:
No objection. I have no strong position on the exact fraction of the screen. Upon measuring the webapp's ratio more carefully, it's ~70%.